test/e2e/config/docker.yaml

```diff
@@ -19,6 +19,8 @@ images:
   loadBehavior: tryLoad
```

Comment:
Better not in this PR, but I think we should rename this file in a follow-up to something which is independent of CAPD. I would suggest "config.yaml"

In the same follow-up we should consider changing our DOCKER_ variables to something more neutral and then use them for CAPIM and CAPD

Reply:
@ykakarap I assume you'll create a follow-up issue for that